components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenManagementDAOImpl.java

```diff
@@ -520,21 +520,31 @@ public void updateAppAndRevokeTokensAndAuthzCodes(String consumerKey, Properties
                     throw new IdentityOAuth2Exception("New Consumer Secret is not specified.");
                 }
 
+                if (properties.containsKey(OAuthConstants.OAUTH_APP_NEW_STATE)) {
+                    // update the consumer secret of the oauth app when the new app state is available.
```

Comment:
Start the comment with a capital letter. Check other places as well.

Reply:
Edited the comments.